Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -262,7 +285,7 @@ export default class FilterComponent extends Vue {
     z-index: 3;
 }
 .filters-width {
-    width: 175px;
+    width: 225px;
```

Comment:
just make sure the button and dropdown align in width

Reply:
they do